frontend/src/component/common/UserAvatar/UserAvatar.tsx

```diff
@@ -94,19 +119,60 @@ export const UserAvatar = forwardRef<HTMLDivElement, IUserAvatarProps>(
 
         const tooltip = disableTooltip ? undefined : tooltipContent(user);
         if (tooltip) {
+            const { main, secondary } = tooltip;
+
+            if (!src && !user?.imageUrl) {
+                return (
+                    <HtmlTooltip
+                        arrow
+                        describeChild
+                        title={<TooltipMainContent>{main}</TooltipMainContent>}
+                    >
+                        {Avatar}
+                    </HtmlTooltip>
+                );
+            }
+
             return (
                 <HtmlTooltip
                     arrow
                     describeChild
+                    maxWidth={400}
                     title={
-                        <>
-                            <TooltipSecondaryContent>
-                                {tooltip.secondary}
-                            </TooltipSecondaryContent>
-                            <TooltipMainContent>
-                                {tooltip.main}
-                            </TooltipMainContent>
-                        </>
+                        <StyledTooltip>
+                            <ConditionallyRender
+                                condition={Boolean(src || user?.imageUrl)}
```

Comment:
Isn't this condition redundant with the if check above?

Reply:
Also ... are we still using conditionally render? 🤔

Reply:
>Isn't this condition redundant with the if check above?

Yep, it was. Good point.

>Also ... are we still using conditionally render? 🤔

I don't know, are we? 🤷 

I've dropped them in this file in the latest commit.